Jenkinsfile.sdk

```diff
@@ -21,14 +21,9 @@ pipeline {
 					sh 'npx lerna run test --ignore lisk-{framework,commander}'
 					dir('commander') {
 						sh '''
-						# TODO: This should not be necessary
-						npm ci
 						LISK_COMMANDER_CONFIG_DIR=$WORKSPACE/.lisk npm test
 						'''
 					}
-					dir('framework') {
-						sh 'npm test'
```

Comment:
Why are we not running `npm test` in `framework/` any more?

Reply:
I thought we have `sdk-framework` jobs for that, so until we merge them i think we don't need them here